src/a-runtime-console/kubernetes/store/oauth-config-store.spec.ts

```diff
@@ -0,0 +1,286 @@
+import { ErrorHandler } from '@angular/core';
+import { fakeAsync, inject, TestBed } from '@angular/core/testing';
+
+import {
+  Http,
+  HttpModule,
+  Response,
+  ResponseOptions,
+  ResponseType,
+  XHRBackend
+} from '@angular/http';
+
+import {
+  MockBackend,
+  MockConnection
+} from '@angular/http/testing';
+
+import {
+  Observable,
+  Subscription
+} from 'rxjs';
+
+
+import { createMock } from 'testing/mock';
+
+import {
+  Logger,
+  Notification,
+  NotificationType
```

Comment:
It appears you are not using these imports.
`Notification, NotificationType`
Nor `Profile` from below.

Reply:
Nice catch as well. I really need to set some time to find an extension that will suggest unused imports or something else that can automatically do that for me.

Reply:
I've removed them with the latest commit.